hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/NettyRpcConnection.java

```diff
@@ -314,7 +314,9 @@ private void succeed(Channel ch) throws IOException {
         }
 
         private void fail(Channel ch, Throwable error) {
-          failInit(ch, toIOE(error));
+          IOException ex = toIOE(error);
+          LOG.warn("Exception encountered while connecting to the server " + remoteId.getAddress(), ex);
+          failInit(ch, toIOE(ex));
```

Comment:
Should be `failInit(ch, ex)`?

Reply:
Yes, once this is addressed, +1 for the PR

Reply:
Addressed the nit, merging.